agent/src/main/java/com/teamscale/jacoco/agent/upload/teamscale/TeamscaleUploader.java

```diff
@@ -63,7 +62,7 @@ private boolean tryUploading(File xml) {
 					teamscaleServer.partition,
 					EReportFormat.JACOCO,
 					teamscaleServer.message,
-					RequestBody.create(MultipartBody.FORM, xml)
+					RequestBody.create(MultipartBody.FORM, new File(coverageFile.getAbsolutePath()))
```

Comment:
I'd prefer a method `CoverageFile.createFormRequestBody()` so other people don't have to think about how to properly do this without wasting memory